shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -83,27 +90,27 @@
-    public static final String STORAGE_CREATE_COUNT = "segmentstore.storage.create_count";      // Counter
+    public static final String STORAGE_READ_LATENCY = "segmentstore.storage.read_latency_ms";   // Histogram, logger: "hdfs" or "filesystem"
+    public static final String STORAGE_WRITE_LATENCY = "segmentstore.storage.write_latency_ms"; // Histogram, logger: "hdfs" or "filesystem"
+    public static final String STORAGE_READ_BYTES = "segmentstore.storage.read_bytes";          // Counter, logger: "hdfs" or "filesystem"
+    public static final String STORAGE_WRITE_BYTES = "segmentstore.storage.write_bytes";        // Counter, logger: "hdfs" or "filesystem"
+    public static final String STORAGE_CREATE_COUNT = "segmentstore.storage.create_count";      // Counter, logger: "hdfs" or "filesystem"
 
     // Cache (RocksDB) stats
-    public static final String CACHE_INSERT_LATENCY = "segmentstore.cache.insert_latency_ms";   // Histogram
+    public static final String CACHE_INSERT_LATENCY = "segmentstore.cache.insert_latency_ms";   // Histogram, logger: "rocksdb"
     public static final String CACHE_WRITE_BYTES = "segmentstore.cache.write_bytes";            // Counter
-    public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram
+    public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram, logger: "rocksdb"
     public static final String CACHE_READ_BYTES = "segmentstore.cache.read_bytes";              // Counter
     public static final String CACHE_TOTAL_SIZE_BYTES = "segmentstore.cache.size_bytes";        // Gauge
-    public static final String CACHE_GENERATION_SPREAD = "segmentstore.cache.gen";              // Histogram
+    public static final String CACHE_GENERATION_SPREAD = "cache.gen";                           // Histogram, logger: "segmentstore"
 
     // DurableDataLog (Tier1) stats
-    public static final String BK_TOTAL_WRITE_LATENCY = "segmentstore.bookkeeper.total_write_latency_ms";   // Including Queue. Per-container Histogram.
-    public static final String BK_WRITE_LATENCY = "segmentstore.bookkeeper.write_latency_ms";               // Exclusively the write to BK. Per-container Histogram.
-    public static final String BK_WRITE_BYTES = "segmentstore.bookkeeper.write_bytes";                      // Counter
-    public static final String BK_WRITE_QUEUE_SIZE = "segmentstore.bookkeeper.write_queue_size";            // Per-container Histogram
-    public static final String BK_WRITE_QUEUE_FILL_RATE = "segmentstore.bookkeeper.write_queue_fill";       // Per-container Histogram
-    public static final String BK_LEDGER_COUNT = "segmentstore.bookkeeper.bookkeeper_ledger_count";         // Per-container Gauge
+    public static final String BK_TOTAL_WRITE_LATENCY = "segmentstore.bookkeeper.total_write_latency_ms";              // Including Queue. Per-container Histogram. Logger: "bookkeeper"
```

Comment:
So this will keep the double `bookkeeper` string?
As in  `bookkeeper_segmentstore_bookkeeper`?

Reply:
I would understand `bookkeeper_segmentstore` or `segmentstore_bookkeeper`, but why `bookkeeper_segmentstore_bookkeeper`?

Reply:
fixed.